src/cellular/dataUpload.hpp

```diff
@@ -44,9 +44,24 @@ class DataUpload : public Task{
     void exit(void);
 
     private:
+    /**
+     * @brief Stores if data upload state successfully initializes.
+     * Variable storing 1 if system successfully enters data upload state, or 0 if system times out
+     * before entering data upload state.
+     */
     int initSuccess;
     system_tick_t lastConnectTime;
+    /**
+     * @brief Exits data upload state.
+     * Function ends data upload process then enters deep sleep state.
+     */
```

Comment:
This function is not used!  Please don't b.s. the documentation.

Please comment out this function.  Please add the following note:

```
In smartfin-fw2/src/dataUpload::DataUpload::exitState(void), we return based on the water sensor state.  If the system is in the water, we redeploy, otherwise we go to sleep.
```